src/PolyhedralGeometry/Cone/constructors.jl

```diff
@@ -72,7 +73,7 @@ end
 
 
 @doc Markdown.doc"""
-    positive_hull([::Type{T} = fmpq,] R::Union{Oscar.MatElem, AbstractMatrix, SubObjectIterator})
+    positive_hull([::Type{T} = fmpq,] R::MatrixUnion)
 
 A polyhedral cone, not necessarily pointed, defined by the positive hull of the
```

Comment:
Old thing, not part of this PR, so feel free to ignore, but: this is not a polyhedral cone, it is a function returning polyhedral cones...
```suggestion
Return a polyhedral cone, not necessarily pointed, defined by the positive hull of the
```